sdcm/cluster.py

```diff
@@ -1183,7 +1183,7 @@ def download_scylla_repo(self, scylla_repo):
             self.remoter.run('sudo curl -o %s -L %s' % (repo_path, scylla_repo))
             result = self.remoter.run('cat %s' % repo_path, verbose=True)
             verify_scylla_repo_file(result.stdout, is_rhel_like=False)
-            self.remoter.run('sudo apt-get update')
+            self.remoter.run(cmd="sudo apt-get update", ignore_status=True)
```

Comment:
After added the workaround, the apt-get update will success, so we don't need to ignore the return status.